api/python/quilt3/util.py

```diff
@@ -366,6 +369,19 @@ def get_install_location():
         loc = get_from_config('default_local_registry').rstrip('/')
     return loc
 
+
+
+def set_config_value(key, value):
```

Comment:
There's no way we don't already have some function that does this? (I think that's what `config(...)` is supposed to do?)

Reply:
We have `api.config()`, but that's method that needs telemetry on it and the telemetry code needs to be able to set the config value, so it became a circular dependency.